Common/FeatureFlags.swift

```diff
@@ -11,11 +11,14 @@ import Foundation
 let FeatureFlags = FeatureFlagConfiguration()
 
 struct FeatureFlagConfiguration: Decodable {
+    let automaticBolusEnabled: Bool
     let cgmManagerCategorizeManualGlucoseRangeEnabled: Bool
     let criticalAlertsEnabled: Bool
+    let debugFeaturesEnabled: Bool
```

Comment:
I suggest not having this _and_ `allowDebugFeatures`.  It will just confuse things.
```suggestion
```